src/lib/API/api.js

```diff
@@ -487,6 +487,41 @@ export class APIService {
     return this.sharedClient.post(url, payload, options)
   }
 
+  async getOTPLEvents(sender, chainId, otpAddress, from) {
+    // if (chainId === NETWORK_ID.FUSE) {
+
+    // } else {
+    const explorer = Config.ethereum[chainId].explorerAPI
+    log.info('polltest -- getOTPLEvents -->', { sender, chainId, otpAddress, from, explorer })
+    const otplLogs = await this.sharedClient.get('/api', {
+      params: {
+        module: 'logs',
+        action: 'getLogs',
+        fromBlock: from - 10000,
+
+        // offset: 1000, // cannot find records?
+        page: 1,
+        address: otpAddress,
```

Comment:
you are missing the topic0 and topic1 filters. topic0 the id of the event topic1 the sender address

Reply:
ah! okay I see I can filter the query call more by defining the topics.

where can I get the eventId from? @sirpy

Reply:
the easiest way is was shown in the issue.
i've also specified in the issue that you need to filter by topics.

Reply:
ok... I thought the eventHash was related to the send transaction.
now realize its the actual eventHash = paymentWithdrawn event